README.md

```diff
@@ -1,5 +1,9 @@
 # Pingh
 
+[![license][license-badge]](https://www.apache.org/licenses/LICENSE-2.0)
+
+[license-badge]: https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat
```

Comment:
Please turn all links into this form, and move all of them to the bottom of the page.